grasp_generation/scripts/create_meshdata_split_and_run_gcp_experiment.py

```diff
@@ -0,0 +1,98 @@
+import subprocess
+from tap import Tap
+import pathlib
+from typing import List, Dict
+import math
+import random
+import pickle
+import datetime
+
+DATETIME_STR = datetime.now().strftime("%Y-%m-%d_%H-%M-%S")
+EXPERIMENT_DIR_PATH_ON_BUCKET = "experiments"
+EXPERIMENT_DIR_PATH_LOCAL = pathlib.Path("../data/experiments")
+
+ALL_MESHDATA_PATH_ON_BUCKET = "all_meshdata"
```

Comment:
let's make `meshdata` in the bucket just be what `all_meshdata` is here -- I find it helpful to keep data dirs as clean as possible